osu.Game/Overlays/BeatmapSetOverlay.cs

```diff
@@ -91,7 +88,7 @@ public BeatmapSetOverlay()
             info.BeatmapSet.BindTo(beatmapSet);
             comments.BeatmapSet.BindTo(beatmapSet);
 
-            Header.Picker.Beatmap.ValueChanged += b =>
+            Header.HeaderContent.Picker.Beatmap.ValueChanged += b =>
```

Comment:
Again this is out of scope of this pull. This was the state of affairs before as well. It may well need to be refactored but not here.

Reply:
Well, you may have a point about this being out of scope, but still, it's really a simple change that would help in the quality of this (avoid exposing header's content publicly), I would've pushed code myself on these cases if I could.

Reply:
I'm trying to keep this change as small and limited in scope as possible so that it doesn't land in purgatory. Which it may do anyway at the current rate of new submissions, anyway.